### ISSUE TITLE
*args and **kwargs are reported as unused even if added to ignored-argument-names

### ISSUE THREAD

**Issue by karyon** [PyCQA/pylint#2214]
### Steps to reproduce
given this function:
```
def frob(arg, *args, **kwargs):
    pass
```
and this setting
```
ignored-argument-names=arg|args|kwargs
```

pylint will report
```
unused-argument / Unused argument 'args'
unused-argument / Unused argument 'kwargs'
```
which i wouldn't expect. weirdly, adding them to `dummy-variables-rgx` actually silences the warning.

using pylint 1.9.2

**Comment by brycepg** [comment]
Duplicate of #1992

**Comment by brycepg** [comment]
Thanks for the report!
